first_timers/first_timers.py

```diff
@@ -112,52 +101,13 @@ def tweet_issues(issues, creds, debug=False):
         title = issue['title']
 
         if len(title) > allowed_title_len:
-            title = title[: allowed_title_len - 1] + ellipse
+            title = title[:allowed_title_len - 1] + ELLIPSE
         else:
             if 'languages' in issue:
-                language_hashTags = ''.join(
-                    ' #{}'.format(lang) for lang in issue['languages']
-                )
+                language_hashTags = ''.join(' #{}'.format(lang) for lang in issue['languages'])
                 hashTags = hashTags + language_hashTags
 
-            max_hashtags_len = MAX_TWEETS_LEN - \
-                (url_len + 1) - (len(title) + 1)
+            max_hashtags_len = MAX_TWEETS_LEN - (url_len + 1) - (len(title) + 1)
 
             if len(hashTags) > max_hashtags_len:
-                hashTags = hashTags[:max_hashtags_len - 1] + ellipse
-
-        url = humanize_url(issue['url'])
-
-        try:
-            # Encoding here because .format will fail with Unicode characters.
-            tweet = '{title} {url} {tags}'.format(
-                title=title,
-                url=url,
-                tags=hashTags
-            )
-
-            if not debug:
-                api.update_status(tweet)
-
-            tweets.append({
-                'error': None,
-                'tweet': tweet
-            })
-
-            log_info('Tweeted issue: {}'.format(issue['title']))
-        except Exception as e:
-            tweets.append({
-                'error': e,
-                'tweet': tweet
-            })
-
-            log_error('Error tweeting issue: {}'.format(issue['title']))
-            log_error('Error message: {}'.format(str(e)))
-
-    return tweets
-
-
-def limit_issues(issues, limit_len=100000):
-    """Limit the number of issues saved in our DB."""
-    sorted_issues = sorted(issues, key=lambda x: x['updated_at'], reverse=True)
-    return sorted_issues[:limit_len]
+                hashTags = hashTags[:max_hashtags_len - 1] + E
```

Comment:
Seems like you have deleted all the code after this line. Can you revert this please.

Reply:
okayy